.rubocop.yml

```diff
@@ -31,17 +26,15 @@ Style/PercentLiteralDelimiters:
 Layout/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
+Layout/AlignHash:
+  EnforcedColonStyle: table
+  EnforcedHashRocketStyle: table
```

Comment:
there was a mixture of styles in the code base, so I choose one i liked.

Reply:
I think you've earned that :-)